clientgui/BOINCGUIApp.cpp

```diff
@@ -84,6 +84,40 @@ typedef unsigned int (*RsvgErrorQuark) (void);
 typedef void (*RsvgHandleGetPixbuf) (void*);
 #endif
 
+#if SUPPORTDARKMODE && defined(__WXMSW__) && wxCHECK_VERSION(3,3,0)
+// wxWidgets' IsDark() returns false on Win 11 now even if dark mode is enabled
+// so we need to additional function to check the registry for the dark mode setting
+bool IsWindowsInDarkMode() {
```

Comment:
The whole function should be replaced with just the `AreAppsDark()` call as you suggested.